binarytree/__init__.py

```diff
@@ -39,6 +46,15 @@ def _new_node(value):
         return _node_init_func(value)
     return (_node_cls or Node)(value)
 
+def _get_id():
```

Comment:
Better still, i've replaced the entire get_id method with a built in python functionality. https://www.programiz.com/python-programming/methods/built-in/id